lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -83,6 +97,12 @@ def create_payment_method(payment_method, options = {})
         commit(:post, 'payment_methods', post_data, options)
       end
 
+      def new_apple_google_pay_flow(payment_method, options)
```

Comment:
there will inevitably be a `newer` apple/goole pay flow.  Is there a different word we can use to distinguish this flow that isn't `new`

Reply:
We are hoping to remove this in a few months this is just for now. So I think new would apply since its temporary